subsys/bluetooth/host/iso.c

```diff
@@ -235,8 +235,9 @@ static int hci_le_setup_iso_data_path(const struct bt_conn *iso, uint8_t dir,
 	sys_put_le24(path->delay, cp->controller_delay);
 	cp->codec_config_len = path->cc_len;
 	cc = net_buf_add(buf, cp->codec_config_len);
-	memcpy(cc, path->cc, cp->codec_config_len);
-
+	if (cp->codec_config_len) {
+		memcpy(cc, path->cc, cp->codec_config_len);
```

Comment:
Would it perhaps be better to use `path->cc_len` instead of `cp->codec_config_len` here? `cp` is a packed struct, so it's possible that accessing its members generates less efficient code than accessing `path->cc_len`.

Reply:
Yes, you are right, I changed it to `if (path->cc_len)`